Code/RDGeneral/Dict.h

```diff
@@ -71,7 +72,9 @@ class Dict {
       for (size_t i=0; i< other._data.size(); ++i) {
         const Pair & pair = other._data[i];
         if(!hasVal(pair.key)) {
```

Comment:
Not part of this bug, but shouldn't `update()` replace the current value if we already have one?